### ISSUE TITLE
Don't alert when <div> inside <dl>.

### ISSUE THREAD

**Issue by Malvoz** [GoogleChrome/lighthouse#3999]
[As per the W3C spec](https://www.w3.org/TR/html52/grouping-content.html#the-dl-element): The `<dl>` element may contain `<div>` elements, wrapping `<dt>/<dd>` pairs for styling purposes. Lighthouse reports this as an accessibility issue, which it is not.

**Comment by patrickhulce** [comment]
Thanks for the report @Malvoz! Would you mind filing this issue with the appropriate rule, failing elements, and perhaps an example URL in the [axe repo](https://github.com/dequelabs/axe-core)? This is what we use for our accessibility testing and they might be able to help provide a better explanation for you.

**Comment by Malvoz** [comment]
Sorry, duplicate of [axe-core issue #262](https://github.com/dequelabs/axe-core/issues/262)! Thanks @patrickhulce.
